test/cucumber-integration-tests/testCucumberTestsInParallel.js

```diff
@@ -39,4 +40,35 @@ describe('Cucumber integration - parallel running single formatter', function()
         assert.strictEqual(errorOrFailed, true);
       });
   });
+
+  it('testCucumberSampleTests in parallel with multiple step definition files', function() {
+
+    commandMocks.elementText('5cc459b8-36a8-3042-8b4a-258883ea642b', 'xx');
+
+    const source = [
+      path.join(__dirname, './sample_cucumber_tests/parallelWithMultipleDefinition')
+    ];
+
+    const {runTests} = common.require('index.js');
+
+    return runTests({
+      source,
+      parallel: true,
+      verbose: true,
+      timeout: 10,
+      silent: false,
+      tags: ['not @fail'],
+      format: 'progress',
+      config: path.join(__dirname, '../extra/cucumber-config-parallel.js'),
+      'format-options': '',
+      ['retry-interval']: 5,
+      ['persist-globals']: true,
+      ['webdriver-host']: 'localhost',
+      ['start-process']: false,
+      ['webdriver-port']: 10190
+    }, {})
+      .then(errorOrFailed => {
+        assert.strictEqual(errorOrFailed, false);
+      });
+  });
 });
```

Comment:
```suggestion
});

```

Reply:
This is nothing special. But adding a new line at the end of the file is a recommended practice by Git to fix diff issues.